package-lock.json

```diff
@@ -4706,7 +4706,8 @@
         },
         "kind-of": {
           "version": "6.0.2",
-          "resolved": ""
+          "resolved": "https://registry.npmjs.org/kind-of/-/kind-of-6.0.2.tgz",
```

Comment:
I'd leave package-lock nonsense like this out of a pull request with functionality changes.